compliance/collection/nodeinventorizer/caching_inventorizer.go

```diff
@@ -0,0 +1,175 @@
+package nodeinventorizer
+
+import (
+	"encoding/json"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/jsonutil"
+)
+
+const (
+	backoffMultiplier = 2
```

Comment:
+1 - we said we want to try 1.5.

Reply:
Yes, the comment and change got lost in a rebase it seems. Please see my answer in the other thread